source/_addonStore/models/version.py

```diff
@@ -62,7 +62,8 @@ def enableCompatibilityOverride(self):
 		"""
 		from addonHandler import AddonStateCategory, state
 		overiddenAddons = state[AddonStateCategory.OVERRIDE_COMPATIBILITY]
-		assert self.name not in overiddenAddons and self.canOverrideCompatibility
+		assert self.name not in overiddenAddons, f"{self.name}, {overiddenAddons}"

```

Comment:
The single assertion was an 'and' of two conditions. The two assertions replacing this are therefore an 'or'. Is this definitely what you meant?

Reply:
I don't think this is true, both assertions must pass and therefore it's an `and`? If one of these is false, the other assertion would still fail.

Reply:
Yep, you're right, sorry about that.